src/pip/_internal/vcs/subversion.py

```diff
@@ -33,6 +40,22 @@ def should_add_vcs_url_prefix(cls, remote_url):
     def get_base_rev_args(rev):
         return ['-r', rev]
 
+    def get_vcs_version(self):
+        # type: (...) -> Union[LegacyVersion, Version]
+        """Gets the version of the currently installed Subversion client."""
```

Comment:
PEP 8 says the first line should be in the form of a command, e.g. "Return the version..."

Can you also document what the method does in the case that (1) the version string can't be parsed, and (2) SVN isn't installed?